pkg/executor/openstack_test.go

```diff
@@ -0,0 +1 @@
+package executor
```

Comment:
Why have you decided to add this? There are no unit tests for this class anyway and integration tests belong to another directory.

Reply:
I didn't save file so that's why it's empty...